pkgs/servers/nextcloud/default.nix

```diff
@@ -1,7 +1,10 @@
 { stdenv, fetchurl, nixosTests }:
 
 let
-  generic = { version, sha256, insecure ? false }: stdenv.mkDerivation rec {
+  generic = {
+    version, sha256,
+    insecure ? false, extraVulnerabilities ? []
```

Comment:
Shouldn't we maybe rename `insecure` to `eol`? Since the meaning of this flag has been shifted with this PR